gulpfile.js

```diff
@@ -67,9 +68,18 @@ gulp.task('pre-test', function () {
     .pipe(istanbul.hookRequire());
 });
 
-gulp.task('test', ['pre-test'], function () {
+gulp.task('test', ['lint', 'pre-test'], function () {
   return gulp.src(argv.file ? argv.file : 'test/test*.js', {read: false})
     // gulp-mocha needs filepaths so you can't have any plugins before it
     .pipe(mocha())
     .pipe(istanbul.writeReports());
-})
+});
+
+gulp.task('lint', function() {
+  return gulp.src('lib/**').pipe(eslint({
+    'rules':{
+        'quotes': [1, 'single'],
+        'semi': [1, 'always']
```

Comment:
I also recommend `comma-dangle: 'always-multiline'` and `quote-props: 'as-needed'` (both of which would entrain changes to this object literal).